Album/ViewControllers/MapControllers/GoogleMapViewController.m

```diff
@@ -161,14 +239,29 @@ - (void)didPost {
 
 // In a storyboard-based application, you will often want to do a little preparation before navigation
 - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
-    // Get the new view controller using [segue destinationViewController].
-    // Pass the selected object to the new view controller.
     if ([segue.identifier isEqual:@"composeSegue"]){
         ComposeViewController *composeVC = [segue destinationViewController];
         composeVC.placeName = self.infoMarker.title;
         composeVC.coordinate = self.infoMarker.position;
         composeVC.placeID = self.infoMarker.snippet;
         composeVC.delegate = self;
+    }else if([segue.identifier isEqual:@"detailsSegue"]){
+        DetailsViewController *detailsVC = [segue destinationViewController];
```

Comment:
I am surprised you did not have to cast.

Reply:
It didn't give me an error haha.